requirements-dev.txt

```diff
@@ -6,6 +6,7 @@ pytest==4.6.9 ; python_version < '3.0'
 pytest==5.3.5 ; python_version >= '3.0'
 pytest-cov==2.8.1
 coveralls==1.11.1
+nose == 1.3.7
```

Comment:
Is this needed?

Reply:
I added this it because the default test suite is configured as nose.collector in setup.py, though it is not necessary to use nose

Reply:
I think we can just drop the test_suite from setup.py? I think that's only needed when trying to run python setup.py test which is deprecated anyway.